### PR TITLE
send payment: federation / muxed addressese changes

### PR DESCRIPTION
**Pull request by acharb**
- hide the memo for Muxed Addresses (slack convo [here](https://stellarfoundation.slack.com/archives/C016CPSLVLG/p1648070784448519))
- display the muxed or federation address instead of its G Address
- send transaction to muxed addresses (so needed to update stellar-sdk)

### CODE REVIEW

File: extension/src/popup/views/Account/index.tsx [hunk 1]
```diff
@@ -111,26 +110,23 @@ export const Account = () => {
...
           </div>
           <div className="AccountView__send-receive-display">
-            <div
-              className="AccountView__send-receive-button"
-              onClick={() => navigateTo(ROUTES.viewPublicKey)}
-            >
-              <span className="AccountView__qr-icon">
-                <Icon.QrCode />
-              </span>
+            <div className="AccountView__send-receive-button">
```
Comment (acharb) [inline extension/src/popup/views/Account/index.tsx:113]: also changed these icons to be NavButtons

File: extension/src/popup/styles/global.scss [hunk 2]
```diff
@@ -35,6 +35,7 @@ a {
 
 #root {
   position: relative;
+  height: 100%;
```
Comment (acharb) [inline extension/src/popup/styles/global.scss:38]: the PopupWrapper needed this to be full height (so can center content correctly). Looks like it lost it when we removed [this wrapper](https://github.com/stellar/freighter/pull/405/files#diff-d9ab29cfb513dab891d9a6d041b202d627e8895ec8d8dbaad90514e1eab61f5dL97-L99)

Comment (piyalbasu) [reply extension/src/popup/styles/global.scss:38]: Good catch! I removed this, looked around, and things *appeared* to be fine. Classic 🤦

File: extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx [hunk 3]
```diff
@@ -166,16 +175,24 @@ export const TransactionDetails = ({
...
+          {federationAddress ? (
+            <div>
+              <span>{federationAddress}</span>
+            </div>
+          ) : (
+            <div className="TransactionDetails__identicon">
+              <IdenticonImg publicKey={destination} />
+              <span>{truncatedPublicKey(destination)}</span>
+            </div>
+          )}
```
Comment (piyalbasu) [inline extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx:187]: I see that this is very similar to 
```
      {federationAddress ? (
        <div>
          <span>{federationAddress}</span>
        </div>
      ) : (
        <div className="SubmitResult__identicon">
          <IdenticonImg publicKey={destination} />
          <span>{truncatedPublicKey(destination)}</span>
        </div>
      )}
```

We should consolidate this repeated logic into a reusable component. I know the styles aren't exactly the same for each, but there's a good amount of overlap. Looks like it's just adding a border to `SubmitResult`. It'd be nice to reuse this as a I can see needing to use this whenever we're dealing with federation addresses.

Comment (acharb) [reply extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx]: good point, moved it to basics with a new sendPayment folder https://github.com/stellar/freighter/pull/409/commits/5f85e7682491eb14d2ed2f0cf955da1417c5e0b2

**Review by piyalbasu — APPROVED** [review]

**Pull request merged**